lgc/patch/NggPrimShader.cpp

```diff
@@ -7005,11 +7034,23 @@ void NggPrimShader::processSwXfbWithGs(ArrayRef<Argument *> args) {
         Value *primData =
             readPerThreadDataFromLds(m_builder.getInt32Ty(), uncompactedPrimitiveIndex,
                                      PrimShaderLdsRegion::PrimitiveData, Gfx9::NggMaxThreadsPerSubgroup * i);
+        unsigned windingIndices[3];
+        if (m_pipelineState->getRasterizerState().provokingVertexMode == ProvokingVertexFirst) {
```

Comment:
The same as the above. Add explanative comments.

Reply:
Done.